src/modules/service-provider/api/provider.controller.ts

```diff
@@ -48,6 +53,36 @@ export class ProviderController {
         return response;
     }
```

Comment:
Is there any reason for this code to still exist?

Reply:
Maybe requirements have changed? The ticket mentions creation of that endpoint.

Reply:
Missed that when reading the ticket, so no changes needed